packages/generators/utils/tooltip.ts

```diff
@@ -7,17 +6,19 @@
  * @returns {Object} An Object that consists of tooltip methods to be invoked
  */
 
-module.exports = {
-	uglify: _ => {
+export default {
+	cssPlugin: (): string => {
 		return `/*
- * We've enabled UglifyJSPlugin for you! This minifies your app
- * in order to load faster and run less javascript.
+ * We've enabled MiniCssExtractPlugin for you. This allows your app to
+ * use css modules that will be moved into a separate CSS file instead of inside
+ * one of your module entries!
  *
- * https://github.com/webpack-contrib/uglifyjs-webpack-plugin
+ * https://github.com/webpack-contrib/mini-css-extract-plugin
  *
  */`;
 	},
-	splitChunks: _ => {
+
+	splitChunks: (): string => {
```

Comment:
void

Reply:
What to pass while calling the method?

https://github.com/webpack/webpack-cli/blob/617af0bddb5140c07b1a0079b377311b10dd027d/packages/generators/init-generator.js#L381

Reply:
_?: void

Reply:
Can we have some kind of eslint rule about this underscore? I keep see inconsistency and honestly I don't like it. But this is a project and I don't mind its rules but where is the rule for this underscore? Developers don't know why (me included) it's there.